### ISSUE TITLE
Node build fails with latest version [Bug]

### ISSUE THREAD

**Issue by kingso** [skills/deploy-to-azure#39]
**Describe the bug**
Building the image with action fails, changed to specific Node v16 and seemed to run.

**Comment by ZakariaAitAli** [comment]
in `.github/workflows/deploy-staging.yml` I changed node version for "Step 4: Deploy to a staging environment based on labels" to work

 ```
    steps:
      - uses: actions/checkout@v3
      - uses: actions/setup-node@v3
        with: 
          node-version: 16
```

**Comment by rffontenelle** [comment]
This was fixed in #59, as pointed out in issue #54
